examples/rust/get_started/examples/04-udp-transport-responder.rs

```diff
@@ -11,10 +12,12 @@ async fn main(ctx: Context) -> Result<()> {
     let udp = UdpTransport::create(&ctx).await?;
 
     // Create a UDP listener and wait for incoming datagrams.
-    udp.listen("127.0.0.1:4000").await?;
+    // Use port 4000, unless otherwise specified by command line argument.
+    let port = std::env::args().nth(1).unwrap_or_else(|| "4000".to_string());
+    udp.listen(format!("127.0.0.1:{port}")).await?;
```

Comment:
let's change this to fixed ports as well

Reply:
@mrinalwadhwa done